### ISSUE TITLE
Provisioning fails when running first time on clean CentOS7 machine

### ISSUE THREAD

**Issue by vchepeli** [gshipley/installcentos#91]
When running installation if failed first time
![capture1](https://user-images.githubusercontent.com/1727278/46871423-aadb0580-ce31-11e8-92e7-e8fa8bac4f84.PNG)
Console was not available

After second run everything was fine and console was available in browser

**Comment by vchepeli** [comment]
@ryannix123 I am using Hyper-V on WIN10 machine. 8GBs RAM and 4CPUs. Every time I run fresh installation it fails

**Comment by vchepeli** [comment]
@ryannix123 trying 3.11 but no luck, have issues with rpms
![capture4](https://user-images.githubusercontent.com/1727278/46875516-29d53b80-ce3c-11e8-979c-bdd18ad154cb.PNG)

**Comment by iamenderst** [comment]
3.11? Not yet. Fresh install this morning...
`# yum info origin-node
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: mirror.team-cymru.com
 * extras: mirrors.ocf.berkeley.edu
 * updates: mirror.sfo12.us.leaseweb.net
Available Packages
Name        : origin-node
Arch        : x86_64
Version     : 3.10.0
Release     : 1.el7.git.0.0c4577e
Size        : 4.2 M
Repo        : centos-openshift-origin
Summary     : Origin Node
URL         : https://github.com/openshift/origin
License     : ASL 2.0
Description : Origin Node
`

**Comment by vchepeli** [comment]
Also tried today VMware Player 15, same error
![capture5](https://user-images.githubusercontent.com/1727278/46949662-78741700-d082-11e8-87f3-a463d6bf9dde.PNG)

**Comment by oveits** [comment]
Which ansible version are you running? Note, that openshift-ansible does not yet support 2.7.0 and I have seen that ansible 2.7.0 is installed in case of CentOS epel-release. Ansible v2.6.5 should work, though.

See https://github.com/openshift/openshift-ansible/issues/10368 for details.

I have created a hotfix on my fork https://github.com/oveits/installcentos on branch hotfix/control-plane-pods-didnt-come-up based on v3.10 of the current repo. Would you be so kind and test my hotfix? If it works, I will create a pull request.

The hotfix has following content:
```
diff --git a/install-openshift.sh b/install-openshift.sh
index b0bac01..b0f31d6 100755
--- a/install-openshift.sh
+++ b/install-openshift.sh
@@ -80,8 +80,16 @@ if [ $? -eq 1 ]; then
        systemctl enable NetworkManager
 fi

+# remove existing Ansible versions:#
+yum remove -y ansible
+
 # install the packages for Ansible
-yum -y --enablerepo=epel install ansible pyOpenSSL
+curl -O http://cbs.centos.org/kojifiles/packages/ansible/2.6.5/1.el7/noarch/ansible-2.6.5-1.el7.noarch.rpm
+# note: epel is needed to install dependencies like python-keyczar
+yum -y --enablerepo=epel install ansible-2.6.5-1.el7.noarch.rpm
+# exclude ansible from automatic updates:
+cat /etc/yum.conf | grep -v -q 'exclude=ansible' && echo "exclude=ansible" >> /etc/yum.conf
+yum -y --enablerepo=epel install pyOpenSSL

 [ ! -d openshift-ansible ] && git clone https://github.com/openshift/openshift-ansible.git

```
Basically, the hotfix is removing ansible, installing ansible v2.6.5 and makes sure that ansible is not updated accidentially.
